Composer/packages/client/config/env.js

```diff
@@ -89,7 +89,8 @@ function getClientEnvironment(publicUrl) {
         GIT_SHA: getGitSha()
           .toString()
           .replace('\n', ''),
-        SDK_PACKAGE_VERSON: '4.6.0-preview2', // TODO: change this when Composer supports custom schema/custom runtime
+        SDK_PACKAGE_VERSION: '4.6.0-preview2', // TODO: change this when Composer supports custom schema/custom runtime
```

Comment:
can you update this to `4.7.0-preview-191208-1` please? We merged updated packages last night into `master`.